packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.less

```diff
@@ -10,14 +10,20 @@
     box-sizing: content-box;
     -webkit-tap-highlight-color: transparent;
     .setCssVariable(color, nui-color-text-default);
-}
+    width: @wizard-label-header-width;
+
+    &.nui-wizard-step-header--completed {
+        .setCssVariable(color, nui-color-text-link);
 
-.nui-wizard-step-header-completed {
-    .setCssVariable(color, nui-color-text-link);
+        .nui-wizard-horizontal-line {
```

Comment:
This class is used in the wizard-horizontal template, so I think it should be moved to the less file for that component.